src/views/auth/CreateNewGame.js

```diff
@@ -32,7 +35,7 @@ class CreateNewGame extends React.Component {
   }
 
   handleGameSessionEndMessage(msg) {
-    this.props.history.push({pathname: '/game-end', state: {usernameWhichHasLeft: msg.hostName, mode:'aborted'}})
+    this.props.history.push({pathname: '/game-end', state: {usernameWhichHasLeft: msg.username, mode:'aborted'}})
```

Comment:
same here, probably better to extract all the handleGameSessionEndMessage() to the InGameView.js, otherwise we have duplicated code